test/unit/io/record_test.cpp

```diff
@@ -15,11 +15,23 @@
 #include <seqan3/io/detail/record.hpp>
 #include <seqan3/core/concept/tuple.hpp>
 #include <seqan3/std/algorithm>
```

Comment:
Can you sort the `seqan3/std` includes in the files you touch?

Reply:
:place sad puppy eyes here:

Thank you for not needing to do this here :)